scripts/install-latest-ci.sh

```diff
@@ -197,14 +197,22 @@ install_application() {
         esac
         ;;
     linux)
-        # Linux doesn't support per-user installs
-        if [[ "$RD_LOCATION" != "system" ]]; then
-            printf "Installing to %s is not supported on Linux; will install into /opt instead.\n" "$RD_LOCATION" >&2
-        fi
-        dest="/opt/rancher-desktop"
-        sudo rm -rf "${dest:?}"
-        sudo unzip -o "$zip_abspath" -d "$dest" >/dev/null
-        sudo chmod 04755 "${dest}/chrome-sandbox"
+        case $RD_LOCATION in
+        system)
+            dest="/opt/rancher-desktop"
+            sudo rm -rf "${dest:?}"
+            sudo unzip -o "$zip_abspath" -d "$dest" >/dev/null
+            sudo chmod 04755 "${dest}/chrome-sandbox"
+            ;;
+        user)
+            dest="$HOME/opt/rancher-desktop"
+            mkdir -p "${dest:?}"
```

Comment:
What is the point of making sure that the directory exists; it will be deleted again by the very next command.

Reply:
It creates `${HOME}/opt`. I actually added that because it otherwise failed.

Reply:
Maybe that should get a comment? And/or it should be moved after the `rm -rf` following it?

Reply:
Comment added (and rebased).  I didn't move it to after `rm -rf` because that means we need to create its parent instead, so doing it before is (extremely slightly) easier.